Components/BarChart/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Bar } from "react-chartjs-2";
+import {} from "chart.js/auto";
```

Comment:
do you need this? Seems like you are not importing anything from chart.js :)

```suggestion
```

Reply:
This is truly a good question. The tutorial that introduced me to chart.js showed me this trick. Apparantly Firefox has some kind of issues w/o this import.

I've just tried it again with this line commented out: works with no difference in chrome, firefox is throwing errors. So I guess I'll just leave it as is.

Reply:
✅